index.html

```diff
@@ -3686,46 +3692,42 @@ <h1>Resolution</h1>
 
     <p>
 This section defines the inputs and outputs of <a>DID resolution</a> and <a>DID
-URL dereferencing</a>. These functions are defined in an abstract way. Their
-exact implementation is out of scope for this specification, but some
-considerations for implementors are discussed in [[?DID-RESOLUTION]].
+URL dereferencing</a>.  Their exact implementation is out of scope for this
+specification, but some considerations for implementors are discussed in
+[[?DID-RESOLUTION]].
     </p>
 
     <p>
 All conformant <a>DID resolvers</a> MUST implement the <a>DID resolution</a>
-functions for at least one <a>DID method</a> and MUST be able to return a <a>DID
-document</a> in at least one conformant <a>representation</a>.
+functions for at least one <a>DID method</a> and MUST be able to produce a
```

Comment:
```suggestion
functions for at least one <a>DID method</a> and MUST be able to return a
```

Propose to leave this unchanged, to avoid confusion with "production" of representations.